packages/react-components/react-progress/src/utils/clampValue.ts

```diff
@@ -0,0 +1,15 @@
+export const clampValue = (value?: number, max: number): number | undefined => {
```

Comment:
I didn't realize at the time that this would give you a build error, it should be defined as:

```suggestion
export const clampValue = (value: number| undefined, max: number): number | undefined => {
```